lib/pmd_translate_checkstyle_format/version.rb

```diff
@@ -1,3 +1,3 @@
 module PmdTranslateCheckstyleFormat
-  VERSION = "0.2.1"
+  VERSION = "0.2.2"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.